scripts/build

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -e
-
+set -x 
```

Comment:
Don't have much experience with our drone stuff, do we usually as `set -x`? If its not the standard seems like its gonna clutter the output a lot